### ISSUE TITLE
Webview2 crashes when calling document.write on newly opened window

### ISSUE THREAD

**Issue by attilamn** [MicrosoftEdge/WebView2Feedback#3491]
I had a similar issue prior to version 101.0.1210.8. 

https://stackoverflow.com/questions/71851863/webview2-crashes-when-calling-document-write-on-newly-opened-window

This problem seems to have come back within the last month, although it doesn't appear on the simple test I created. 

```
private void Wv_CoreWebView2InitializationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2InitializationCompletedEventArgs e)
        {

            if (Deferral != null)
            {
                Args.NewWindow = wv.CoreWebView2;
                Deferral.Complete();
            }
            wv.CoreWebView2.NewWindowRequested += CoreWebView2_NewWindowRequested;
        }
```
Taking the last event handler, it seems to fail here "Args.NewWindow = wv.CoreWebView2;". If I put a break on that line and wait a second to continue, it works fine.

Any idea if this has been identified and when this is going to be fixed? Is there a workaround?

Thanks!

**Comment by attilamn** [comment]
I forgot to add that in the debug output window, I get the following and it appears to be related:
[0516/165541.100:ERROR:(-1)] Check failed: false.

**Comment by novac42** [comment]
Thanks for reporting the issue. So the issue occurred on version prior to 101.0.1210.8, but you are seeing it again with latest runtime and SDK?

**Comment by attilamn** [comment]
Yes. I am currently using runtime 113.0.1774.42 and have tried updating my SDK to 1.0.1774.30. I have tested it with both WPF and Windows Forms, with the same issue occurring. I have seen this on both Windows 10 and 11.

**Comment by attilamn** [comment]
bump

**Comment by plantree** [comment]
Hi @attilamn,

Sorry for my late reply. To help better understand your issue, if you're available, could you provide a simple `.sln` project to reproduce? Thanks!
